project-descriptor.yml

```diff
@@ -0,0 +1,13 @@
+service:
```

Comment:
Is this used by Nebulae to spin up the app?

Reply:
Yep.